streams/streams-scala/src/main/scala/org/apache/kafka/streams/scala/kstream/KStream.scala

```diff
@@ -796,6 +807,7 @@ class KStream[K, V](val inner: KStreamJ[K, V]) {
    * @return a [[KStream]] that contains records with unmodified key and new values (possibly of different type)
    * @see `org.apache.kafka.streams.kstream.KStream#transformValues`
    */
+  @deprecated
```

Comment:
Oy, forgot about the scala API ... again. Do we need to add the new interfaces to the scala API? Also, the scala `deprecated` annotation takes parameters about when it was deprecated and what you should use instead, which we should make use of.

Reply:
Got it. Params are added. I also forgot to add `processValues` to the Scala API. Should be there now.